tests/suite/test_v_s_route_upstream_options.py

```diff
@@ -144,11 +147,11 @@ def test_when_option_in_v_s_r_only(self, kube_apis,
          ["max_fails=3 ", "fail_timeout=33s ", "max_conns=0;",
           "proxy_connect_timeout 44s;", "proxy_read_timeout 22s;", "proxy_send_timeout 55s;",
           "keepalive 1024;", 'set $default_connection_header "";',
-          "client_max_body_size 3m;"],
+          "client_max_body_size 3m;", " 100k;"],
```

Comment:
zone size are not configurable in the upstream field of VS/VSR. do we need to test for them here?

Reply:
I wasn't sure if to add this check or not, I'll remove it then.

Reply:
I agree this may be misleading (the same check exists in `test_virtual_server_configmap_keys`) as we don't support this for VS/VSR. I'll remove them then.